src/ios/classes/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -26,14 +26,14 @@
 #import "AppDelegate+SalesforceHybridSDK.h"
 #import "UIApplication+SalesforceHybridSDK.h"
 #import "InitialViewController.h"
-#import "SFLocalhostSubstitutionCache.h"
-#import "SFHybridViewConfig.h"
-#import "SalesforceSDKManager.h"
-#import "SFPushNotificationManager.h"
-#import "SFSDKAppConfig.h"
-#import "SFDefaultUserManagementViewController.h"
-#import "SFSDKLogger.h"
-#import "SalesforceSDKManagerWithSmartStore.h"
+#import <SalesforceHybridSDK/SFLocalhostSubstitutionCache.h>
```

Comment:
Now that we use dynamic libs, we don't have to update the header imports anymore.